Tools/Scripts/webkitdirs.pm

```diff
@@ -842,6 +846,7 @@ sub argumentsForConfiguration()
     push(@args, '--maccatalyst') if (defined $xcodeSDKPlatformName && $xcodeSDKPlatformName eq 'maccatalyst');
     push(@args, '--32-bit') if ($architecture eq "x86");
     push(@args, '--gtk') if isGtk();
+    push(@args, '--haiku') if isHaiku();
```

Comment:
Putting this between GTK and WPE seems slightly surprising, especially given this doesn't appear to be alphabetical?

Reply:
I went with sort-of alphabetical, putting it after GTK but not reordering what was already there. I can move it elsewhere, I guess at the end of the list after playstation then?

Reply:
Yeah, I guess? It's vaguely grouped by groups of ports, as far as I can tell. `--32-bit` and `--jsc-only` are certainly in odd places here, but otherwise that seems to hold true.